src/DataFixtures/ORM/LoadReferentManagedUserData.php

```diff
@@ -25,7 +25,7 @@ public function load(ObjectManager $manager)
             'original_id' => $this->getReference('adherent-1')->getId(),
             'email' => $this->getReference('adherent-1')->getEmailAddress(),
             'postal_code' => $this->getReference('adherent-1')->getPostalCode(),
-            'city' => $this->getReference('adherent-1')->getCity(),
+            'city' => $this->getReference('adherent-1')->getCityName(),
```

Comment:
`getCity()` matches `address_city_insee`, but script SQL which fills the `projection_referent_managed_users` table takes values of `address_city_name`.